core/http_bench.rs

```diff
@@ -164,15 +169,9 @@ impl Behavior for HttpBench {
 }
 
 fn main() {
-  let js_source = include_str!("http_bench.js");
-
   let main_future = lazy(move || {
-    let mut isolate = deno_core::Isolate::new(HttpBench());
+    let isolate = deno_core::Isolate::new(HttpBench());
 
-    // TODO currently isolate.execute() must be run inside tokio, hence the
-    // lazy(). It would be nice to not have that contraint. Probably requires
-    // using v8::MicrotasksPolicy::kExplicit
```

Comment:
Please retain this TODO - but move it above the definition of `deno_core::Isolate::execute()`